agr_literature_service/api/tests/test_018_topic_entity_tag.py

```diff
@@ -0,0 +1,240 @@
+import pytest
+from fastapi import HTTPException
+from pydantic import ValidationError
+from sqlalchemy import MetaData, create_engine
+from sqlalchemy.orm import sessionmaker
+from sqlalchemy.orm.exc import NoResultFound
+
+from agr_literature_service.api.crud.topic_entity_tag_crud import (
+    create, show, patch, destroy
+)
+from agr_literature_service.api.database.config import SQLALCHEMY_DATABASE_URL
+from agr_literature_service.api.database.base import Base
+from agr_literature_service.api.models import (TopicEntityTagModel,
+                                               TopicEntityTagPropModel)
+from agr_literature_service.api.schemas import (
+    TopicEntityTagSchemaCreate,
+    TopicEntityTagSchemaUpdate)
+from agr_literature_service.api.crud.mod_crud import create as mod_create
+from agr_literature_service.api.crud.user_crud import create as user_create
+from agr_literature_service.api.crud.reference_crud import create as reference_create
+from agr_literature_service.api.schemas import ReferenceSchemaPost
+
+metadata = MetaData()
+
+engine = create_engine(SQLALCHEMY_DATABASE_URL, connect_args={"options": "-c timezone=utc"})
+SessionLocal = sessionmaker(bind=engine, autoflush=True)
+db = SessionLocal()
+
+# Add tables/schema if not already there.
+Base.metadata.create_all(engine)
+
+# Exit if this is not a test database, Exit.
+if "literature-test" not in SQLALCHEMY_DATABASE_URL:
+    exit(-1)
+
+fb_mod = None
```

Comment:
fb_mod is initialized and updated on mod_create, but doesn't seem part of the test.  This is fine by me.

Reply:
I may remove this. I just copied and pasted from another initialisation script that needed mods.